src/ciadmin/generate/worker_pools.py

```diff
@@ -609,6 +609,13 @@ def update_config(config, name, attributes):
             else:
                 del config["worker-manager-config"]["launchConfigId"]
 
+        if attributes.get("scratch-disks", 0) > 0:
+            scratch_disks = [attributes["scratch-disk-definition"]] * attributes[
+                "scratch-disks"
+            ]
+            for instance in config.get("instance_types", []):
+                instance["disks"] = instance.get("disks", []) + scratch_disks
+
```

Comment:
TBH I'm not super happy with this approach to add scratch disks per variant, but couldn't think of a better way